fenix/app/src/main/res/values/strings.xml

```diff
@@ -2306,4 +2306,10 @@
     <string name="a11y_action_label_read_article">read the article</string>
     <!-- Action label for links to the Firefox Pocket website. Talkback will append this to say "Double tap to open link to learn more". -->
     <string name="a11y_action_label_pocket_learn_more">open link to learn more</string>
+    <!-- Action label for the switch element when it is off. Talkback will say, for example "Automatic translation switch off".-->
+    <!-- "Automatic translation" will be the text from the switch element. -->
+    <string name="accessibility_off_switch">Switch OFF</string>
+    <!-- Action label for the switch element when it is on. Talkback will say, for example "Automatic translation switch on".-->
+    <!-- "Automatic translation" will be the text from the switch element. -->
+    <string name="accessibility_on_switch">Switch ON</string>
```

Comment:
I was expecting the strings to be `%1$s Switch ON`, but since we added the parameter to the content description strings instead which are in static_string. Let's move these back into static_strings for the time being. What worries me here is that we are now exposing these strings to translations, but they don't see the other side of the strings which are currently in static_strings. I know we included "Automatic translation" here, but without a localization note about what and how "Automatic translation" is used, I don't think there is enough context here for our translators at this moment.